config/config.go

```diff
@@ -558,6 +559,11 @@ func DefaultConfig() (*Config, error) {
 	return &c, nil
 }
 
+// IsLoadBalancingAlgorithmValid Check validity of the provided load balancing algorithm
```

Comment:
Please take a look at [Go Doc Comments](https://go.dev/doc/comment) for recommendations on how to format / write them. Given the expressive nature of the function name I would argue that we can leave out the doc here.

Reply:
Thanks, will do.

Reply:
I specifically had this quote in mind:

> [...] Go doc comments use complete sentences. For a package comment, that means the [first sentence](https://go.dev/pkg/go/doc/#Package.Synopsis) begins with “Package ”.

In this case, that would translate to something like this:

```suggestion
// IsLoadBalancingAlgorithmValid checks the validity of the provided load balancing algorithm.
```

But the sentence doesn't add any value over the name of the function, so it might as well be left out.

Same goes for:

```go
// OverrulePoolLoadBalancingAlgorithm Overwrites the load balancing algorithm of a pool by that of a specified endpoint, if that is valid.
func (p *EndpointPool) OverrulePoolLoadBalancingAlgorithm(endpoint *Endpoint) {
```

Reply:
done